package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "azure-devops-node-api",
     "description": "Node client for Azure DevOps and TFS REST APIs",
-    "version": "12.1.1",
+    "version": "12.2.0",
```

Comment:
The version should be updated in package-lock.json

Reply:
updated!